src/mercury_engine_data_structures/formats/bmbls.py

```diff
@@ -1,9 +1,10 @@
 import construct
 
-from mercury_engine_data_structures.formats import BaseResource, standard_format
+from mercury_engine_data_structures.formats.base_resource import BaseResource
+from mercury_engine_data_structures.formats.standard_format import create
 from mercury_engine_data_structures.game_check import Game
 
-BMBLS = standard_format.create('base::animation::CBlendSpaceResource', 0x02020001)
+BMBLS = create('base::animation::CBlendSpaceResource', 0x02020001)
```

Comment:
I prefer to import the module instead of a function from the module. And this change makes no difference for loops.

Reply:
okay, I reverted that change. surprised that worked since I thought the circular import issue was from calling `from mercury_engine_data_structures.formats import XYZ`